RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -109,15 +109,6 @@
         "version" : "11.6.4"
       }
     },
-    {
```

Comment:
should be reverted ofc... and updated to use Sargon once the needed branch is merged.